### PR TITLE
add basic Diagrm components

### PR DESCRIPTION
**Pull request by raylrui**
## diagram components
 
1. add basic structure for each node, include nodes, edges
2. add status variable for each node
3. set styling for different status
4. add workflow initial constants

example:
overview:
image

wgs:
image

wts:
image

Refactor components code about import format.

### CODE REVIEW

**Comment by pdiakumis** [comment]
That looks amazing:-)

**Comment by victorskl** [comment]
React Flow, Pete.! :)

**Comment by raylrui** [comment]
Also combine the Refactor components code about " import"  format. 
Most components now has no default import except Layout components. @williamputraintan

File: src/components/dialogs/SimpleDiag.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import { FC } from 'react';
```
Comment (williamputraintan) [inline src/components/dialogs/SimpleDiag.tsx:1]: Just a minor typo in the filemane

**Review by williamputraintan — APPROVED** [review]
Looks really good!

File: src/components/dialogs/SimpleDiag.tsx (hunk 1 above)
Comment (raylrui) [reply src/components/dialogs/SimpleDiag.tsx]: Fixed, great thanks, merging

**Pull request merged**